azure-pipelines.yml

```diff
@@ -97,10 +97,12 @@ jobs:
     name: ''
     displayName: Install Signing Plugin
     enabled: True
+    condition: eq('${{ parameters.SignTypeToUse }}', 'real')
```

Comment:
☕️💡 thought share: Since this signin part was alsonpartnofnone bug against the repo, I wonder if we don’t differentiate between test and real for sign in so that test can be as close to real and tested.

what do you think @hsubramanianaks and @elenavillamil ! ☕️☕️🕵️‍♀️

Reply:
Guidance from David was that PR builds should not have signing (for security reasons to avoid signing something sent from someone outside our team since this is a public repo - specially after this pipeline is approved for real signing). Plan is to have one build pipeline for PRs and one "official" build pipeline with signing for release. This is step one of refactoring. Once this works I am working on pipeline that fully removes signing that will be used with PRs. This pipeline will be unlinked from PRs and always have true signing (potentially run it of main once a day). PR description has details as well. Let me know if this doesnt address the concern.

Reply:
As a note on the linked issue, it looks like the problem was that a test signed build actually passed the signing validation check and was accidentally shipped with a VSCode extension release. Test signing in these pipelines doesn't actually sign with a valid trusted certificate and sign check validation should ideally fail for a test signed build as well as an unsigned build.

Reply:
resolving based on offline sync, feel free to re-open if needed.